### PR TITLE
Support msg deleting from chats

### PR DESCRIPTION
**Pull request by lightme16**
- improve some function namings
- allow to delete selected msg from the chat by `d` keypress
- update view to handle situation when there is a chat with no message(all were deleted)

further ideas:
- when sending delete request to the telegram, API responds back with `updateDeleteMessages`. in order to be more robust, we could remove message from UI only whem ACK from telegram side was received in form of the `updateDeleteMessages` message.
- message removal is pretty destructive operation so we may want to also add user conformation before msg removal.

### CODE REVIEW

File: tg/controllers/__init__.py [hunk 1]
```diff
@@ -60,6 +62,10 @@ def handle_msgs(self):
             elif keys == 'G':
                 if self.model.jump_bottom():
                     self.refresh_msgs()
+            elif keys == 'd':
```
Comment (paul-nameless) [inline tg/controllers/__init__.py:65]: Can we use dd, like in vim.
This way we will reduce accidental msg deletion and make it more user friendly (vim friendly)?

File: tg/models/__init__.py [hunk 2]
```diff
@@ -213,7 +224,33 @@ def get_msgs(self, chat_id, offset=0, limit=10):
...
+        if chat_id is None:
+            return False
+        selected_msg = self.current_msgs[chat_id]
+        msg_item = self.msgs[chat_id].pop(selected_msg)
+        self.current_msgs[chat_id] = min(
+            selected_msg, len(self.msgs[chat_id]) - 1
+        )
+        log.info(f"Deleting msg from the chat {chat_id}: {msg_item}")
+        message_ids = [msg_item["id"]]
+        r = self.tg.delete_messages(chat_id, message_ids, revoke=True)
```
Comment (paul-nameless) [inline tg/models/__init__.py:251]: Maybe we could try to delete msg first and if success delete locally?

File: tg/controllers/__init__.py (hunk 1 above)
Comment (lightme16) [reply tg/controllers/__init__.py]: done

File: tg/models/__init__.py (hunk 2 above)
Comment (lightme16) [reply tg/models/__init__.py:251]: Yep, we could try to listen for `updateDeleteMessages`. So, we could use it for messages removal. 
But first, let's merge https://github.com/paul-nameless/tg/pull/11/files#diff-0b1338b3fa65880d711319687a8f2a45R204 into master, so `remove_message`  function will be available in the codebase

Comment (lightme16) [reply tg/models/__init__.py:251]: So, let's handle it in the next PR

**Pull request merged**